app/controllers/idv/enter_password_controller.rb

```diff
@@ -81,6 +81,18 @@ def step_indicator_step
       :get_a_letter
     end
 
+    def self.step_info
```

Comment:
I thought we weren't going to do the steps after hybrid handoff? If we aren't going to use them would you be okay stashing them on a branch somewhere instead of checking them in unused?

Reply:
Stashed on [dprice-lg-11039-flow-policy-future-steps](https://github.com/18F/identity-idp/tree/dprice-lg-11039-flow-policy-future-steps) and I'll add a commit removing them from this branch.

We ended up doing the steps through DocumentCapture and LinkSent, similar to where the browser back button works.

Reply:
Resolved in e9d71d170fa9cab74f30d7665f5ba63c9c0ec043